01_05_Conditions.txt

```diff
@@ -4,92 +4,170 @@ Conditions
+exception caused by the overinflation of their fear fuzzy logic field. 
+
+    A Character experienceing Greater Fear can take no actions, except rolling 
+to defeat fear. DC 75, Will save or 5 turns, whichever happens first.
 
 == Lonely ==
 
     Being Lonely is a condition most commonly contracted from having 1 Charisma
 and failing to successfully socialize, or by trying and failing to socialize 
-with an ally who has 1 Charisma. You haven't met Mazlow's Hierarchy of needs, 
-and this affects your performance. While Lonely, you move 1m slower every two 
-actions (round down) and take a -5 to skill rolls. 
+with an ally who has 1 Charisma. A Lonely Character hasn't met Mazlow's 
+Hierarchy of needs, and this affects their performance. 
+
+    A Lonely Character moves 1m slower every two Actions (round down) and takes 
+a -5 to skill rolls. 
 
 == Necrosis ==
 
-    2d10 +5 damage per turn. Seek medical attention immediately. If left 
-unhealed for more than a day, the infection spreads 1 foot per day. Medicine DC 
-75, must be rolled.
+    Necrosis is a buildup of dead organic tissue in a biologic Character, 
+usually caused by gangrene or biological weapons. Typically, any hair, fur or
+scales fall off of the affected area and the flesh underneath turns black and
+dies.
+
+    A Character aflicted with Necrosis takes 2d10 +5 damage per Round at the 
+begining of their turn. If left unhealed for more than a day, the infection
+spreads 1 foot per day. A Medicine check of DC 75 must be rolled to cure it, 
+usually by surgically removing the dead tissue and generously applying healing 
+biofoam.
 
 == Pinned Down ==
 
-    Target pinned (hunkered, no movement) for 2 full turns or until they pass a 
-Will saving throw at DC of 70, whichever is quicker.
+    A target that is behind cover and is either pinned by the threat of sniper
+fire or a continuous stream of bullets must overcome the psychological instinct
+to stay put and not get shot.
+
+    A target that is pinned is hunkered and cannot move for 2 full Rounds or 
+until they pass a DC 70 Will Save, whichever is quicker.
 
 == Plasma Burn ==
 
-    1d10+1 stackable damage per turn, for 2 turns if the target's armor is 
-pierced/missed. If the targets armor is hit/pierced, the armor's AP Level is 
-reduced by 1, permanently (repairable, DC 60% Repair check), not stackable. 
+    Many weapons in the Compound X Universe fire bolts of burning plasma, which
+have the tendency to stick to a target and their armor, continuing to do damage
+even after the initial impact of the projectile.
+
+    A Character that has a Plasma Burn takes 1d10 + 1 Plasma Damage per Plasma
+Burn Token per round at the beginning of their turn. Each "token" lasts for 2 
+turns if the target's armor was pierced or missed. 
+
+    If the targets armor is hit/pierced, the armor's Armor Piercing Level is
+reduced by 1, permanently (repairable, DC 60% Repair check). While the damage 
```

Comment:
Edit stockpile: inconsistent with the description for plasma damage. There, the APL reduction ends after the encounter instead of permanent until repair. We should pick one duration and use it both places for ease of memory.